tests/conftest.py

```diff
@@ -83,3 +90,26 @@ def app(versionfile, docker_env_vars, disable_migrations, disable_log_output):
     """Needed for pytest-flask."""
     app = create_app(versionfile.strpath)
     return app.app
+
+
+@pytest.fixture
+def db(app):
+    """Reset database for each test."""
+    with app.app_context():
+        _db.init_app(app)
+        _db.create_all()
+        # we just created.
+        yield _db
+        _db.session.remove()
+        _db.drop_all()
+
+
+@pytest.fixture
+def s3():
```

Comment:
I think this fixture also needs to depend on the `docker_env_vars` fixture because it uses the `PATCH_BUCKET_NAME` env var.